homeassistant/components/reolink/strings.json

```diff
@@ -1,6 +1,6 @@
 {
   "config": {
-    "flow_title": "{short_mac} ({ip_address})",
```

Comment:
Should we remove the short mac?

Reply:
Personally I like having the mac to identify devices, but we also have the IP adress, so I don't mind to much removing it.

Do note that the hostname is not always very descriptive.
For my E1 Zoom it is nicely "Reolink E1 zoom"
But for my NVR is is just "Reolink" which does not tell you much if you have multiple reolink devices.

@frenck shall I remove it or leave it...

Reply:
For now I have removed the short_mac, I can revert the commit if needed.

Reply:
I think it is good to remove, for the average user it looks like gibberish.